ovos_plugin_common_play/ocp/base.py

```diff
@@ -1,4 +1,10 @@
 from ovos_plugin_common_play.ocp.settings import OCPSettings
+from ovos_plugin_common_play.ocp.status import MediaState, PlayerState, TrackState
```

Comment:
Consider renaming this to `ocp_backend` and `mycroft_cps` to `mycroft_backend` to make it more clear they are both backends

Reply:
hum, is this comment in the right place? no sure how it applies

`ovos_plugin_common_play.ocp` is the right name, `ovos_plugin_common_play` is the plugin itself, `ocp` is the media player lib used by the plugin, didn't make sense to split the repos but those are basically independent modules. 

Due to the plugin expected directory format the plugin code needs to be in the top level `__init__` file

You can run `ocp` module directly without the plugin, even as a system service if wanted